### ISSUE TITLE
best accuracy: 0.000 after train 200000 steps

### ISSUE THREAD

**Issue by tienduchoang** [clovaai/deep-text-recognition-benchmark#51]
My dataset is image contain 14 number characters (for [example](https://drive.google.com/drive/folders/1pUiWY6PZwvNWCSX46vvD3g9QCfKH7yTk)). I config my opt like below:
```
character: 0123456789-
sensitive: False
PAD: True
data_filtering_off: False
Transformation: TPS
FeatureExtraction: ResNet
SequenceModeling: BiLSTM
Prediction: Attn
num_fiducial: 20
input_channel: 3
output_channel: 512
hidden_size: 256
num_gpu: 1
num_class: 13

```
but when i train 200.000 step, the accuracy still 0%.

**Comment by sungraepark** [comment]
@tienduchoang 
How about the train loss curve?

**Comment by zobeirraisi** [comment]
I actually got the same result after 100000 steps?

**Comment by ku21fan** [comment]
Hello, 

Can you upload your `opt.txt` file?
I recommend checking 2 parts.
1. If you use adam optimizer with `--adam` option, you should use lower learning rate, such as `--lr 0.001`.
2. data filtering part in [here](https://github.com/clovaai/deep-text-recognition-benchmark/blob/master/dataset.py#L130-L148). 
If your validation image has the character which is not in `opt.character` (in this case `0123456789-`), the image would be filtered. 

Best
